app/views/service_providers/_form.html.erb

```diff
@@ -1,20 +1,20 @@
 <%= form.error_notification %>
 <fieldset class='usa-fieldset'>
 
-  <legend class='usa-sr-only'>Production Configuration</legend>
+  <legend class='usa-sr-only'>Dashboard Configuration</legend>
   <%= form.label :prod_config,
-                 label: 'Production Configuration',
-                 class: 'usa-label',
-                 hint: I18n.t('service_provider_form.prod_config').html_safe %>
```

Comment:
[question] do we still want a hint?

Reply:
What does the hint do? I couldn't find any difference bw leaving and removing it.

Reply:
I also don't see any documentation for a `hint` attribute with a quick SE search

Reply:
Hint is the mouse hover alt text I believe.

Reply:
Also accessibility screen reader stuff

Reply:
it looks like the hint SHOULD be getting wrapped with a div with the uswds class `usa-hint` according to this code: https://github.com/18F/identity-dashboard/blob/main/config/initializers/simple_form.rb#L26.

Reply:
ok @ajfarkas and i did a little more investigation -- it looks like we sometimes duplicate the `hint` attribute, both on the `form` and the `label`. 

the `label hint: I18n.t('service_provider_form.prod_config').html_safe` version creates a `hint` label on the attribute which is not a standard attribute and doesn't seem to do anything (and all the information is being rendered in more standard ways via the form anyways), so i believe it's safe to remove. we should also go through in another commit and do some cleanup by removing the `hint` from the labels to reduce confusion in the future -- that can be on a different PR if you'd like to move this forward @ajfarkas